client/src/main/scala/com/dwolla/cloudflare/domain/model/ratelimits/Implicits.scala

```diff
@@ -1,11 +1,12 @@
 package com.dwolla.cloudflare.domain.model.ratelimits
 
 import com.dwolla.cloudflare.domain.dto.ratelimits._
+import com.dwolla.cloudflare.domain.model.tagRateLimitId
 
 private[cloudflare] object Implicits {
   implicit def toModel(dto: RateLimitDTO): RateLimit = {
     RateLimit(
-      id = dto.id.get,
+      id = tagRateLimitId(dto.id.get),
```

Comment:
I know you didn't introduce this in this PR, but do you happen to know whether and/or why this `.get` is safe?

Reply:
I think its optional on the DTO because that is used on create (when not set) or get/list (when it is set). I guess its only converted to a model on get/list, or times when we know there is an id? I guess there could be another DTO object for create that might clear it up?

Reply:
Ok, thanks. This kind of thing feels like a likely source of bugs, and I've seen it repeated in a couple spots. Don't need to worry about it here—I'll keep thinking about it.